### PR TITLE
Closes #1889 - Update Set-Service examples to reflect available Start…

### LINKED ISSUE #1889: Set-Service examples incorrectly claim that Get-WmiObject is needed to determine a service's startup type
https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Set-Service.md claims with respect to its example commands:

> These commands use the Get-WmiObject cmdlet to get the Win32_Service object for the service, because the ServiceController object that Get-Service returns does not include the start mode.

However, what `Get-Service` returns - at least as of Windows PowerShell v5.1 / PowerShell Core v6.0.0-rc - _does_ have a `.StartType` property - it's just that the default formatting information does not show it (consider adding it).

```powershell
# The property is clearly there.
> (Get-Service wuauserv).StartType
Manual

# However, it isn't displayed by default.
> Get-Service wuauserv | Format-List
# ... no StartType 

# You must ask for it by name or use *
> Get-Service wuauserv | Format-List *
# ...
StartType           : Manual
# ...
```

Therefore, the example should be revised to use `Get-Service` instead.

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

See below re what versions may or may not be impacted.

### PR DESCRIPTION
**Pull request by bobbytreed**
…Type property

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [X] Impacts 6.0 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.1/Microsoft.PowerShell.Management/Set-Service.md) | :x:Error | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Management%2FSet-Service.md)
[]() | :x:Error | | [Details](#user-content-)

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Management%2FSet-Service.md"></a>
### [reference/5.1/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.1/Microsoft.PowerShell.Management/Set-Service.md)
 - **[Error]** Unable to load file: 5.1/Microsoft.PowerShell.Management/Set-Service.md via processor: AzureCliDocumentProcessor: :295:(1253) &#39;```

### -WhatIf

Shows what would happen if t...&#39;
 Invalid yaml: expected simple key-value pairs

<a name=""></a>
### []()
 - **[Error]** Markdown.MAML.Transformer.HelpSchemaException: :295:(1253) &#39;```

### -WhatIf

Shows what would happen if t...&#39;
 Invalid yaml: expected simple key-value pairs
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.GetInnerContexts(DocumentBuildParameters parameters, IEnumerable`1 processors, TemplateProcessor templateProcessor, IHostServiceCreator creator)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.BuildCore(DocumentBuildParameters parameter, IMarkdownServiceProvider markdownServiceProvider, BuildInfo currentBuildInfo, BuildInfo lastBuildInfo)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.Build(IList`1 parameters, String outputDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument(BuildJsonConfig config, TemplateManager templateManager, String baseDirectory, String outputDirectory, String pluginDirectory, String templateDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument()

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806291409570478-2634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=TMmJvgx7MxAnvaJGrORTtdljJm%2FqRPLItOC5dYrQwUE%3D&st=2018-06-29T14%3A12%3A41Z&se=2018-07-30T14%3A17%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/3.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-3.0&branch=pr-en-us-2634) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/4.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-4.0&branch=pr-en-us-2634) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-5.0&branch=pr-en-us-2634) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.1/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-5.1&branch=pr-en-us-2634) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/6/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-6&branch=pr-en-us-2634) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806291547524007-2634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FXUkLMH7RqR4Wdghre3%2BLIxT3eUUsH7%2FYAQjDG0DOjk%3D&st=2018-06-29T15%3A54%3A29Z&se=2018-07-30T15%3A59%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/3.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-3.0&branch=pr-en-us-2634) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/4.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-4.0&branch=pr-en-us-2634) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-5.0&branch=pr-en-us-2634) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.1/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-5.1&branch=pr-en-us-2634) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/6/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-6&branch=pr-en-us-2634) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806291603115990-2634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NTmoTRBfOTMy2WMfutu2oq6Ply2iWMKDzzCBYy%2BdlAA%3D&st=2018-06-29T16%3A08%3A40Z&se=2018-07-30T16%3A13%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bobbytreed** [comment]
#sign-off

**Comment by bobbytreed** [comment]
#sign-off

File: reference/3.0/Microsoft.PowerShell.Management/Set-Service.md [hunk 1]
```diff
@@ -7,137 +7,135 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=113399
...
 Starts, stops, and suspends a service, and changes its properties.
+
 ## SYNTAX
 
 ### Name (Default)
+
 ```
-Set-Service [-ComputerName <String[]>] [-Name] <String> [-DisplayName <String>] [-Description <String>]
- [-StartupType <ServiceStartMode>] [-Status <String>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
+Set-Service [-Name] <string> [-ComputerName <string[]>] [-DisplayName <string>] [-Description <string>]
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Set-Service.md]: .NET class names should be capitalized - <String>

File: reference/3.0/Microsoft.PowerShell.Management/Set-Service.md [hunk 2]
```diff
@@ -7,137 +7,135 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=113399
...
 
-The second command uses the Get-WmiObject cmdlet to get an instance of the WMI Win32_Service that represents the Task Scheduler service.
-The first element in the command gets all instances of the Win32_service class.
+```powershell
+Set-Service -Name "winrm" -Status Running -PassThru
+```
 
-The pipeline operator (|) passes the result to the Where-Object cmdlet, which selects instances with a value of "Schedule" in the Name property.
+This command starts the WinRM service.
+The command uses the Status parameter to specify the desired status, which is running, and the *PassThru* parameter to direct `Set-Service` to return an object that represents the WinRM service.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Set-Service.md]: https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/3-STYLE-GUIDE.md#formatting-syntax-elements

Property and parameter names should be in **bold** - eg **PassThru**

File: reference/3.0/Microsoft.PowerShell.Management/Set-Service.md [hunk 3]
```diff
@@ -7,137 +7,135 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=113399
...
 Specifies one or more computers.
 The default is the local computer.
 
 Type the NetBIOS name, an IP address, or a fully qualified domain name of a remote computer.
-To specify the local computer, type the computer name, a dot (.), or "localhost".
+To specify the local computer, type the computer name, a dot (.), or localhost.
 
 This parameter does not rely on Windows PowerShell remoting.
-You can use the ComputerName parameter of Set-Service even if your computer is not configured to run remote commands.
+You can use the *ComputerName* parameter even if your computer is not configured to run remote commands.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Set-Service.md]: Bold the parameter name

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/3.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-3.0&branch=pr-en-us-2634) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/4.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-4.0&branch=pr-en-us-2634) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.0/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-5.0&branch=pr-en-us-2634) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/5.1/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-5.1&branch=pr-en-us-2634) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Set-Service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1889/reference/6/Microsoft.PowerShell.Management/Set-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/set-service?view=powershell-6&branch=pr-en-us-2634) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806291803151140-2634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=f5zv4wLYCUj8QakXU3RjWm0tvapwr0hodi9hR95YSYE%3D&st=2018-06-29T18%3A08%3A40Z&se=2018-07-30T18%3A13%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**